pkg/skaffold/watch/triggers.go

```diff
@@ -43,6 +46,8 @@ func NewTrigger(opts *config.SkaffoldOptions) (Trigger, error) {
 		return &pollTrigger{
 			Interval: time.Duration(opts.WatchPollInterval) * time.Millisecond,
 		}, nil
+	case "notify":
```

Comment:
This should be added to the description of the `trigger` command line flag

Reply:
right